Intrepid.podspec

```diff
@@ -25,8 +25,8 @@ Pod::Spec.new do |s|
   s.source        = { :git => "https://github.com/IntrepidPursuits/swift-wisdom.git", :tag => "#{s.version}" }
   s.exclude_files = "tests/**/*"
   s.platform      = :ios
-  s.ios.deployment_target = "9.0"
```

Comment:
We just moved this up to 9.0 from 8.0, do we really want to drop 9.0 support already?